### ISSUE TITLE
Ruby 2.7 warnings in Execute resource

### ISSUE THREAD

**Issue by tas50** [chef/chef#9735]
```
    * execute[set hostname to chef-bk-ci.chef.io] action run/opt/chef/embedded/lib/ruby/gems/2.7.0/gems/chef-16.0.245/lib/chef/provider/execute.rb:52: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/opt/chef/embedded/lib/ruby/gems/2.7.0/gems/chef-16.0.245/lib/chef/mixin/shell_out.rb:54: warning: The called method `shell_out!' is defined here
```

**Comment by tas50** [comment]
@lamont-granquist fixed this

**Comment by skukla** [comment]
Forgive my ignorance, but has the fixed been released yet?  I still encounter it using Chef Infra Client 16.0.257.

**Comment by tas50** [comment]
This fix has not yet been released. Most likely we'll ship a bug fix for 16.0 next week which will include the fix in https://github.com/chef/chef/pull/9762
